docs/mkdocs.yml

```diff
@@ -926,6 +929,8 @@ nav:
       - Memory:
           - ./api_reference/memory/chat_memory_buffer.md
           - ./api_reference/memory/index.md
+          - ./api_reference/memory/simple_composable_memory.md
+          - ./api_reference/memory/vector_memory.md
```

Comment:
@logan-markewich: not sure if I did this right, or if this anything else required to add the api ref for the new objects

Reply:
This is correct, but we also need to create these md files too

Reply:
ah okay, 1/2 marks. Do we have a process to create them automatically?

Reply:
For integrations yes, for core stuff no (the integrations are setup in a way that's very easy to automate, core not so much)

Reply:
cool, no worries. It just copied the `.md` file we had for `chat_memory_buffer` and created new ones for `vector_memory` and `simple_composable_memory`. It looked easy to do .... a one-liner — hope i did it right lol.